### PR TITLE
Issue #SH-614 fix: Course completion Message is not displayed when user consumes a video for 20% and the content has a Green Tick and course Progress is 100%

### PR DESCRIPTION
**Pull request by Nik720**

### CODE REVIEW

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts [hunk 1]
```diff
@@ -200,6 +200,10 @@ export class CoursePlayerComponent implements OnInit, OnDestroy {
       .subscribe(courseProgressData => {
         this.courseProgressData = courseProgressData;
         this.progress = courseProgressData.progress ? Math.floor(courseProgressData.progress) : 0;
+        if (this.activatedRoute.snapshot.queryParams.showCourseCompleteMessage === 'true') {
+          this.showCourseCompleteMessage = this.progress === 100 ? true : false;
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts:204]: Check for progress == 100 or >100, I some cases, it is coming as >100%

File: src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts [hunk 2]
```diff
@@ -59,6 +59,7 @@ export class AssessmentPlayerComponent implements OnInit, OnDestroy {
   totalContents = 0;
   consumedContents = 0;
   layoutConfiguration;
+  isCourseCompletionPopupShown = false;
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts:62]: onCourseCompleteClose() {
    this.showCourseCompleteMessage = false;
  }

This was not added here. But the same is added in course-player-component

File: src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts (hunk 1 above)
Comment (Nik720) [reply src/app/client/src/app/modules/learn/components/course-consumption/course-player/course-player.component.ts]: Updated

File: src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts (hunk 2 above)
Comment (Nik720) [reply src/app/client/src/app/modules/learn/components/course-consumption/assessment-player/assessment-player.component.ts:62]: onCourseCompleteClose method is already available

**Pull request merged**